Update DraftConstitution.md objects

craig9 opened the pull request.

Commit:
Update DraftConstitution.md

ghost commented:
looks good to me!

ghost commented:
Michael mentioned in the conversation that the model rules list of objects was a very detailed list, as though if it's not in the list, we can't do it.. Hence my attempt (in the chat) at a plain English version of the list. I'll make a new commit to this pull request branch for further comment.

Commit:
More changes to the associations objects

The model rules seems to be an exhaustive list of what the association can do. Almost as if - if it's not on the list, we can't do it. So we're trying to cover all bases. But in plain english, not legalese. I ignored the investments part, since I don't think that will be an issue. If it becomes one, we'll have an SGM, and add something to cover it.
